styleguide/components/21-atoms/list/_list.scss

```diff
@@ -46,7 +46,7 @@ ul {
 
   &.iconlist-small {
     li {
-      padding-left: 1.4rem;
+      padding-left: 1.6em;
```

Comment:
Update line 34 and you can remove this one

Reply:
Done :)